Dockerfile

```diff
@@ -1,14 +1,12 @@
-FROM golang:1.19-alpine as builder
+FROM golang:1.21-alpine as builder
 
-ARG caexporter_version=0.0.2
-
-RUN apk --no-cache add git
+RUN apk --no-cache add git make
 
 WORKDIR /opt/caexporter
 COPY . .
-RUN go build -ldflags="-X github.com/prometheus/common/version.Version=${caexporter_version}} -X github.com/prometheus/common/version.Revision=$(git rev-parse HEAD) -X github.com/prometheus/common/version.Branch=$(git branch --show-current)" ./
+RUN make build
 
-FROM alpine:3.16
+FROM alpine:3.19
```

Comment:
We might want to consider running this on `busybox` or maybe a distroless image base, should reduce the image size and attack surface

Reply:
change din 547685b